src/components/Modals/Onboarding.vue

```diff
@@ -194,7 +193,7 @@
 <script>
 import reqsData from '@/requirements/typed-requirement-json';
 
-const placeholderText = 'Select one';
+const placeholderText = ['Select one'];
```

Comment:
Any reason why the placeholder text includes brackets? Looks a bit off on the onboarding modal